components/server/ee/src/container-module.ts

```diff
@@ -71,6 +72,7 @@ export const productionEEContainerModule = new ContainerModule((bind, unbind, is
     bind(GithubAppRules).toSelf().inSingletonScope();
     bind(PrebuildStatusMaintainer).toSelf().inSingletonScope();
```

Comment:
**issue**: Adding this here although probably not related to this piece of code. Is it known that _Branches_ and _Prebuilds_ are not listed after adding a proejct?

Reply:
`Branches` should be. 
`Prebuilds` are not expected in this change-set.

Reply:
OK! For some projects branches load but showing `UNKNOWN COMMIT MESSAGE` while others fail to load branches with the following error in the console:

```js
Uncaught (in promise) Error: Request getProjectOverview failed with message: Cannot read property 'avatar_url' of undefined
```

| Projecr Branches List |
|-|
| <img width="894" alt="Screenshot 2021-08-17 at 10 24 54 AM" src="https://user-images.githubusercontent.com/120486/129682300-57c3c3fb-c75b-48a4-accd-3f7b852a085c.png"> |

Reply:
`UNKNOWN COMMIT MESSAGE` that's a placeholder.

Reply:
Oh, ok! 🙈